jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -93,18 +93,21 @@ public boolean errorPageForMethod(String method)
     }
 
     /**
-     * Write an error response.
+     * Write an error response, or signal that the error will be handled by a down stream handler.
      * <p>
-     * In Servlet implementations of the {@link ErrorHandler}, this method is overridden to signal that
-     * a sendError should be triggered a when it enters the servlet channel.
+     * The default implementation calls {@link Response#writeError(Request, Response, Callback, int)} and returns {@code true}.
      * <p>
+     * <p>
+     * Servlet implementations will override this method to trigger a sendError when the {@code ServletChannel} is started and return {@code false}.
+     * </p>
```

Comment:
```suggestion
     * <p>
     * {@code ErrorHandler} extensions may override this method to modify the state of the 
     * {@code request} and/or {@code response} so that when the method returns {@code false} 
     * an error page will be generated by a subsequent {@code Handler}.  For example, 
     * Servlet implementations override this method to set attributes that trigger a 
     * Servlet `sendError` call when the {@code ServletHandler} invokes the {@code ServletChannel}.
     * </p>
```